packages/compass-web/src/entrypoint.tsx

```diff
@@ -117,6 +117,7 @@ function CompassWorkspace({
   return (
     <WorkspacesProvider
       value={[
+        // TODO: we probably need a welcome workspace here?
```

Comment:
Do we add multi-connections support to compass-web already? Should this also be the default there? You can only see CompassWeb in the sandbox right now if you're already connected or is work there further along than I realise?

Reply:
For now CompassWeb still have the render tree for the single connection flow. But having the CompassWelcome as one of the provided workspace there shouldn't be a problem because its only provided as a value but can only be used when something opens it, which is noone for now.

Reply:
I'll leave it out for now because I see MyQueries is also missing from there anyway.